### ISSUE TITLE
Fallback album cover does not display on the "Now Playing" screen.

### ISSUE THREAD

**Issue by BigBoyBarney** [harmonoid/harmonoid#448]
**Describe the bug**
Fallback album cover does not display on the "Now Playing" screen.

**To Reproduce**
1. Import album with cover.jpg in the folder
2. Enable fallback album cover
3. Play the track

**Expected behavior**
Fallback album cover shows everywhere instead of the placeholder one.

**Screenshots**
![image](https://github.com/harmonoid/harmonoid/assets/86712892/b044a6f2-2679-4662-bfb9-9cb4dc27ba16)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version [e.g. v0.3.8]
 
Thanks in advance, I love this app a lot already!

P.S.: To add a small feature request on this, it would be super cool to have the album cover as the background for the now playing screen.

**Comment by alexmercerind** [comment]
Have you enabled fallback album arts in the settings?

**Comment by BigBoyBarney** [comment]
Yes, I assume that's why it works almost everywhere, like the album browser, just not in the now playing tab.
![image](https://github.com/harmonoid/harmonoid/assets/86712892/83b0bca7-6be8-466f-8465-4c5073663b86)

**Comment by MithNo1** [comment]
Yes I'm facing the same bug please fix this!

**Comment by MithNo1** [comment]
I found a weird fix for this issue but still this is a big hassle so please fix it fully.

The way to fix with current version is you just manually apply the cover art in a album song even that does not work for some reason but you just upload the cover and hit save then go full screen with the track song you just applied your cover art to then force close the app then reopen it. Then it works. This on PC version.

As you can see this is big hassle. But I wanted to point it out for your help. 

And also for some reason the cover art quality decreases in full screen and becomes blur a lot for some reason so please look into that as well.

**Comment by alexmercerind** [comment]
Thanks for the feedback! I just happen to be bit busy, I will resume work sometime in future.

**Comment by MithNo1** [comment]
cool! I just recently discovered your app. It's a shame PC does not have music player with great UI likes yours. It's just amazing to think you have worked on this alone! Like It looks that it is built by a team of many! Lol! hoping to see these bugs fixed soon whenever you resume back! Take care :) ❤️

**Comment by alexmercerind** [comment]
Try out the latest update please.
